src/index.css

```diff
@@ -1,7 +1,262 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
```

Comment:
- [OPTIONAL ]  According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please make these in order to stick to the original design:
- Kindly add an icon to the right of your navigation bar 😜 
- What if we reduce the space between the `book-genre` and `book-title` 😜 
- Let's add this missing vertical bar to separate the container class div with the rightblock class div

- Your Design:

![Screenshot](https://user-images.githubusercontent.com/79058364/172908067-17d6bd6c-f19d-4df9-9c6c-00ec4fc7758a.png)


- Figma design

![bookstore](https://user-images.githubusercontent.com/79058364/162426330-10521c8c-526f-4fca-9450-dc3154d41b55.png)